dom.bs

```diff
@@ -1253,6 +1253,9 @@ for discussion).
 
  <li><p>Set <var>event</var>'s {{Event/eventPhase}} attribute to {{Event/NONE}}.
 
+ <li><p>Set <var>event</var>'s {{Event/target}} attribute to null if {{Event/target}}'s root is a
+ <a for=/>shadow root</a> and <var>event</var>'s <a>composed flag</a> is unset.
```

Comment:
> and <var>event</var>'s <a>composed flag</a> is unset.

We should remove this condition.
We need to clear event.target even if composed flag is set because target and relatedTarget can be identical in the middle and event path can be *trimmed* for that case.

Reply:
Good catch. So we can determine if the event is stopped at shadow root just by looking at whether target's root is a shadow root.